vscode-extensions/project-explorer/extension/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "vscode-project-explorer-extension",
+  "private": true,
+  "version": "11.2.0",
+  "description": "The Axon Ivy Project Explorer is meticulously designed to seamlessly integrate Axon Ivy projects into your development workflow.",
+  "displayName": "Axon Ivy Project Explorer",
+  "publisher": "AxonIvy",
+  "author": "Axon Ivy AG",
+  "homepage": "https://developer.axonivy.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/axonivy/vscode-extensions"
+  },
+  "keywords": [
+    "vscode-extension",
+    "diagram",
+    "axonivy",
+    "ivyteam",
+    "process",
+    "bpmn",
+    "engine"
+  ],
+  "preview": true,
+  "icon": "assets/ivy-logo.png",
+  "engines": {
+    "vscode": "^1.54.0"
+  },
+  "activationEvents": [
+    "onStartupFinished"
+  ],
+  "main": "./lib/extension.js",
+  "devDependencies": {
+    "@types/vscode": "^1.54.0",
+    "ts-loader": "^9.4.2"
+  },
+  "scripts": {
+    "build": "tsc -p ./",
```

Comment:
I think it must be changed soon to a bundler, if we have common stuff on other packages ...

Reply:
agree, but the switch is not a big deal